### DISCUSSION TITLE
Your application cannot be reviewed until you fix the following: Your uploaded proof has already been used previously.

### DISCUSSION THREAD

**Discussion by SaktiKalyanJena** [community/community#113200] [GitHub Education]
### Hi everyone,

I am applying for student pack. But it's throwing error: 

**

> Your application cannot be reviewed until you fix the following:
> Your uploaded proof has already been used previously.

**

**Comment by lin20260407** [comment]
Maybe your student account has already been requested?

You might want to consider changing your student account @edu.

**Comment by GCustom** [comment]
Mine did this, too, after telling me to fix my billing info, so I gave it a different screenshot than the one I initially gave them, since the complaint was that blank, blank didn't match First Last on my document, not that the doc was bad.

**Comment by samus-aran** [comment]
Hi @SaktiKalyanJena 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [Your uploaded proof has already been used previously.](https://github.com/orgs/community/discussions/111352#uploaded-proof-already-used)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
